src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/prisonData/PrisonRegisterClient.kt

```diff
@@ -14,16 +13,8 @@ class PrisonRegisterClient(
   @Autowired private val prisonRegisterWebClient: WebClient
 ) {
 
-  fun getAllPrisons(): Mono<List<Prison>> {
-    return prisonRegisterWebClient
-      .get()
-      .uri("/prisons")
-      .retrieve()
-      .bodyToMono(object : ParameterizedTypeReference<List<Prison>>() {})
```

Comment:
We're gonna need this in my change shortly to expose prisons endpoint on ref-data controller :)